HousingRepairsOnlineApi/Helpers/RepairDayWindowHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public class RepairDayWindowHelper : IRepairDayWindowHelper
+    {
+        private IEnumerable<RepairDayWindow> _repairDayWindows;
+
+        public RepairDayWindowHelper(IEnumerable<RepairDayWindow> repairDayWindows)
+        {
+            this._repairDayWindows = repairDayWindows;
+        }
+
+        public int GetDaysForRepair(Repair repair)
+        {
+            Guard.Against.Null(repair, nameof(repair));
+
+            var result = 0;
+            if (_repairDayWindows != null && _repairDayWindows.Any())
+            {
+                result = _repairDayWindows.First(r => r.Priority == repair.Priority).NumberOfDays;
```

Comment:
Using `First` will fail if a `Priority` is not found, which should be handled here (potentially be using `FirstOrDefault` and checking for `!= null` before getting `NumberOfDays`).